Gemfile.lock

```diff
@@ -690,6 +695,13 @@ GEM
       rack (< 2.1.0)
       rack-protection (>= 1.5.0)
       redis (>= 3.3.5, < 5)
+    sidekiq-ent (1.8.1)
+      einhorn (= 0.7.4)
+      sidekiq (>= 5.2.3)
```

Comment:
Is the sidekiq stuff here a dup of: https://github.com/department-of-veterans-affairs/vets-api/pull/4327/files

Reply:
Rebased and all right now i think